Core/uWebshop.Domain/Model/OrderDTO/Order.cs

```diff
@@ -44,7 +44,7 @@ public class Order : IConvertibleToOrderInfo
 	    public DateTime? FulfillDate;
 		public PaymentInfo PaymentInfo;
 		public int PaymentProviderPrice;
-		public int? PaymentProviderOrderPercentage;
+		public decimal? PaymentProviderOrderPercentage;
```

Comment:
Do you really need to use percentages with more than 2 decimal places? If not, please keep using the same "philosophy" of having a percentage represented in an `int` * 100